package.json

```diff
@@ -32,16 +32,18 @@
   },
   "homepage": "http://gpii.net",
   "dependencies": {
+    "ajv": "^1.4.9",
+    "dedupe-infusion": "1.0.0",
     "first-discovery": "git+https://github.com/GPII/first-discovery.git",
```

Comment:
I've filled a [JIRA](https://issues.gpii.net/browse/GPII-1519) and [pull request](https://github.com/GPII/first-discovery/pull/107) for adding the "gpii-" prefix to the first discovery tool.